extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -234,6 +239,23 @@ export class SubscriptionService {
 
       user().assert(this.pageConfig_, 'Page config is null');
 
+      if (this.viewer_.hasCapability('auth')) {
+        const serviceIds = ['local'];
+        this.platformStore_ = new PlatformStore(serviceIds);
+        this.viewer_.sendMessageAwaitResponse('auth', dict())
+            .then(entitlementData => {
+              dev().assert(typeof entitlementData == 'object',
```

Comment:
The `entitlementData` should be an object with `authorization` field that will contain a string value - the signed JWT string. For now, we just need to decode JWT and take the payload. Very similar to https://github.com/subscriptions-project/swg-js/blob/master/src/runtime/entitlements-manager.js#L219

Reply:
I guess, Done